qt/widgets/instrumentview/inc/MantidQtWidgets/InstrumentView/Shape2D.h

```diff
@@ -301,6 +302,60 @@ class Shape2DRing : public Shape2D {
   double m_yWidth;
 };
 
+/**
+ * A sector: area defined by an inner and an outer arcs of circle, interrupted
+ * at angles startAngle and endAngle.
+ *
+ * The constructor takes a center QPoint property, 2 angles, and 2 widths for
```

Comment:
widths? Make clear how the angles are stored in radians or degrees. Also make a note the both start and end angles are supposed to 0->2pi and not -pi->pi.